### DISCUSSION TITLE
Download old version of imagemagick

### DISCUSSION THREAD

**Discussion by zmdyy0318** [ImageMagick/ImageMagick#4899] [Help]
Where can I download ImageMagick_7.1.0-24-H8-x64-dll.exe and ImageMagick_7.1.0-24-H8-x86-dll.exe.
I can not found it on imagemagick website.

**Comment by dlemstra** [comment]
We don't keep every old version on our ImageMagick server. I would advise you to upgrade to the most recent version.
